addons/mail/static/src/components/activity/activity.xml

```diff
@@ -47,12 +47,14 @@
                             </div>
                         </t>
                         <a
+                            href="#"
                             class="o_Activity_detailsButton btn py-0"
                             t-att-class="activityView.areDetailsVisible ? 'text-primary' : 'btn-link btn-primary'"
+                            t-att-aria-expanded="activityView.areDetailsVisible ? 'true' : 'false'"
                             t-on-click="activityView.onClickDetailsButton"
                             role="button"
                         >
-                            <i class="fa fa-info-circle" role="img" title="Info"/>
+                            <i class="fa fa-info-circle" role="img" title="Info" aria-label="Info"/>
```

Comment:
Not necessarily a problem, but this was added only in this version?

Reply:
Yes, it was intentional. This and the below cases were the only differences. I did it this way because they were actually missing on previous versions.

If you prefer consistency with previous versions, you can remove it.